### PR TITLE
Documentation for downloading requirements with docker containers.

### PR DESCRIPTION
**Pull request by seriva**

### CODE REVIEW

File: docs/home/howto/CLUSTER.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ or VMs and should meet the following requirements:
 5. A requirements machine that:
     - Runs the same distribution as the air-gapped cluster machines/VMs (RedHat 7, CentOS 7, Ubuntu 18.04)
     - Has access to the internet.
+   If you dont have access to a similar machine/VM with internet access you can also try and download the requiments with a Docker container. More information [here](./CLUSTER.md#downloading-offline-requirements-with-a-docker-container).
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
   If you don't have access to a similar machine/VM with internet access you can also try and download the requiments with a Docker container. More information [here](./CLUSTER.md#downloading-offline-requirements-with-a-docker-container).
```

File: docs/home/howto/CLUSTER.md [hunk 2]
```diff
@@ -1020,3 +1021,77 @@ specification:
   managed: true
 provider: azure
 ```
+
+## Downloading offline requirements with a Docker container
+
+This paragraph describes how to use a Docker container to download the requirements for airgapped/offline installations. At this time we don't offically support this and we still recommand using a full distribution which is the same as the air-gapped cluster machines/VMs.
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]: "air-gapped" or "air gapped"

```suggestion
This paragraph describes how to use a Docker container to download the requirements for air-gapped/offline installations. At this time we don't officially support this and we still recommend using a full distribution which is the same as the air-gapped cluster machines/VMs.
```

File: docs/home/howto/CLUSTER.md [hunk 3]
```diff
@@ -1020,3 +1021,77 @@ specification:
...
+
+A few points:
+- This only describes how to setup the the Docker containers for downloading. The rest of the steps are similar as in the paragraph [here](./CLUSTER.md#how-to-create-an-epiphany-cluster-on-existing-air-gapped-infrastructure).
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
- This only describes how to setup the Docker containers for downloading. The rest of the steps are similar as in the paragraph [here](./CLUSTER.md#how-to-create-an-epiphany-cluster-on-existing-air-gapped-infrastructure).
```

File: docs/home/howto/CLUSTER.md [hunk 4]
```diff
@@ -1020,3 +1021,77 @@ specification:
...
+
+### Ubuntu 18.04
+
+For Ubuntu you can use the following command to launch a container:
+
+```shell
+docker run -v /shared_folder:/home <--platform linux/amd64 or --platform linux/arm64> --rm -it ubuntu:18.04
+```
+
+As the ```ubuntu:18.04``` image is multi-arch you can include ```--platform linux/amd64``` or ```--platform linux/arm64``` to run the container as the specified architecture. The ```/share_folder``` should be a folder on your local machine containing the requirement scripts.
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
As the ```ubuntu:18.04``` image is multi-arch you can include ```--platform linux/amd64``` or ```--platform linux/arm64``` to run the container as the specified architecture. The ```/shared_folder``` should be a folder on your local machine containing the required scripts.
```

File: docs/home/howto/CLUSTER.md [hunk 5]
```diff
@@ -1020,3 +1021,77 @@ specification:
...
+
+### RedHat 7.x
+
+For RedHat you can use the following command to launch a container:
+
+```shell
+docker run -v /shared_folder:/home <--platform linux/amd64 or --platform linux/arm64> --rm -it registry.access.redhat.com/ubi7/ubi:7.9
+```
+
+As the ```registry.access.redhat.com/ubi7/ubi:7.9``` image is multi-arch you can include ```--platform linux/amd64``` or ```--platform linux/arm64``` to run the container as the specified architecture. The ```/share_folder``` should be a folder on your local machine containing the requirement scripts.
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
As the ```registry.access.redhat.com/ubi7/ubi:7.9``` image is multi-arch you can include ```--platform linux/amd64``` or ```--platform linux/arm64``` to run the container as the specified architecture. The ```/shared_folder``` should be a folder on your local machine containing the requirement scripts.
```

File: docs/home/howto/CLUSTER.md [hunk 6]
```diff
@@ -1020,3 +1021,77 @@ specification:
...
+
+For running the ```download-requirements.sh``` script you will need a RedHat developer subscription to register the running container and make sure you can access to offical Redhat repos for the packages needed. More information on getting this free subscribtion [here](https://developers.redhat.com/articles/getting-red-hat-developer-subscription-what-rhel-users-need-know).
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
For running the ```download-requirements.sh``` script you will need a RedHat developer subscription to register the running container and make sure you can access to official Redhat repos for the packages needed. More information on getting this free subscription [here](https://developers.redhat.com/articles/getting-red-hat-developer-subscription-what-rhel-users-need-know).
```

File: docs/home/howto/CLUSTER.md [hunk 7]
```diff
@@ -1020,3 +1021,77 @@ specification:
...
+```shell
+docker run -v /shared_folder:/home --platform linux/arm64 --rm -it arm64v8/centos:7.9.2009
+```
+
+x86_64:
+```shell
+docker run -v /shared_folder:/home --platform linux/amd64 --rm -it amd64/centos:7.9.2009
+```
+
+The ```/share_folder``` should be a folder on your local machine containing the requirement scripts.
```
Comment (plirglo) [inline docs/home/howto/CLUSTER.md]:
```suggestion
The ```/shared_folder``` should be a folder on your local machine containing the requirement scripts.
```

**Review by plirglo — CHANGES_REQUESTED** [review]

**Review by rafzei — APPROVED** [review]

**Review by plirglo — APPROVED** [review]

**Pull request merged**